### ISSUE TITLE
Tool does not match instructions for 'Web-developer-facing change to existing code (PSA)'

### ISSUE THREAD

**Issue by davidben** [GoogleChrome/chromium-dashboard#1686]
**Describe the bug**
The [instructions](https://www.chromium.org/blink/launching-features) for "Web-developer-facing change to existing code (PSA)" say after "Step 0: Create a ChromeStatus entry, and choose your feature type", you are supposed to follow:

> Step 1: Write up motivation and implement code
> Fill out the “Motivation” section with a brief summary, and proceed to the “Implementing” stage in ChromeStatus.  

However, instead, after step 0 I am taken to a page like this:
<img width="528" alt="Screen after step 1" src="https://user-images.githubusercontent.com/109690/150169967-97fff219-afee-4e87-8f21-0445befe100a.png">

There is no "Motivation" field. If I click the most natural Start button (which already doesn't match the instructions, because that's supposed to come before filling out "Motivation"), I get this page, also with no such prompt:

<img width="521" alt="Screen after clicking the Start button" src="https://user-images.githubusercontent.com/109690/150170198-d543478a-808b-4347-a00a-3c9cda63961e.png">

Indeed if I preview all the subsequent stages, I can't find "Motivation" under any of them. I can only find it under "Edit all fields".

**To Reproduce**
1. Try to launch a PSA for a change to existing code
2. Create a new ChromeStatus entry per instructions
3. Try to follow next instructions

**Expected behavior**
The tool matches the instructions.

**Additional context**

**Comment by miketaylr** [comment]
Thanks for filing @davidben, I also find this confusing.

**Comment by davidben** [comment]
Further complication is if I try to use "Edit all fields", it refuses to let me save the "Motivation" entry without also entering something for "Debuggability", even though that doesn't seem to apply for "Web-developer-facing change to existing code (PSA)" features.

**Comment by davidben** [comment]
Another complication is that although the instructions say the dev trial is optional (as it should be for a small PSA-level change), but it's very unclear from the website how one skips this.

**Comment by jpmedley** [comment]
@cwilso I think we need to update the instructions to match the interface. I can do this, but I don't think I have write access to the right pages.

**Comment by davidben** [comment]
The interface itself was a bit confusing independent of the instructions. It was very unclear how to skip a DevTrial for a small PSA.

**Comment by jrobbins** [comment]
I'll add the motivation field.  

I think this mistake happened because for other feature types motivation was part of the incubation phase, but web-developer-facing changes do not have an incubation phase, they go straight to implementation.  So, I will add motivation there.

**Comment by jpmedley** [comment]
Please leave this open until I've at least looked at the documentation. I've learned not to assume that a UI change fixes everything.

**Comment by jpmedley** [comment]
Let me rephrase that. I always approach these things with the assumption that both UI and docs need to be improved.
